src/makeInfinityAdder.js

```diff
@@ -1,12 +1,30 @@
 'use strict';
 
 /***
- *
  * @return {function}
  */
-
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+  let result = 0;
```

Comment:
The 'result' variable seems to be unnecessary. You can directly return 'sum' when no arguments are provided and reset 'sum' to zero after that.